src/lib.rs

```diff
@@ -121,11 +121,10 @@
 //! [window_builder_new]: window::WindowBuilder::new
 //! [window_builder_build]: window::WindowBuilder::build
 //! [window_id_fn]: window::Window::id
-//! [`Event`]: event::Event
-//! [`WindowEvent`]: event::WindowEvent
+//! [`Event::WindowEvent`]: event::WindowEvent
 //! [`DeviceEvent`]: event::DeviceEvent
 //! [`UserEvent`]: event::Event::UserEvent
-//! [`LoopExiting`]: event::Event::LoopExiting
+//! [`Event::LoopExiting`]: event::Event::LoopExiting
```

Comment:
I don't want this PR to turn into a "let's fix _all_ `winit` intradoc links", but a few more changes are still needed to make this consistent.

No clue what the point is with manual relative `.html` links that may not resolve when certain features are not turned on... Pointing the user to a 404 page :(

Reply:
This is probably a problem for another time, but these methods are deprecated anyway.